mlem/contrib/torch.py

```diff
@@ -140,9 +141,11 @@ class TorchModelIO(ModelIO):
 
     def dump(self, storage: Storage, path, model) -> Artifacts:
         self.is_jit = isinstance(model, torch.jit.ScriptModule)
-        save = torch.jit.save if self.is_jit else torch.save
         with storage.open(path) as (f, art):
-            save(model, f)
+            if self.is_jit:
+                torch.jit.save(model, f)
+            else:
+                torch.save(model, f, pickle_module=cloudpickle)
```

Comment:
can you also try `dill` here? since we already use it

Reply:
you mean instead of `cloudpickle`?

Reply:
Yes

Reply:
`dill` didn't work, using `cloudpickle` only